### DISCUSSION TITLE
Batch edit JSON files

### DISCUSSION THREAD

**Discussion by recycledsocks** [EsriPS/arcgis-assistant-feedback#22] [Ideas]
It would be fantastic if someone could please add the ability to find & replace sections of code in multiple JSON files at once.
This would allow bulk updates of service urls, bulk modification of attributes stored in web maps and apps, plus much more!

Thanks!

**Comment by jpeterson** [comment]
Hey @recycledsocks (sorry I missed this for so long). I would find this incredibly useful myself, particularly when needing to promote content in dev > test > prod environments. We will definitely have a look at how this might work and update here when have have something to share.

**Reply by recycledsocks** [reply]
Hey @jpeterson , thanks for your response! I had an urgent requirement so I have started developing my own python script to do what I require. It works off an excel config file and will replace either the entire JSON, or short snippits of code. Have you had any progress? Would be happy to compare approaches, although I only commissioned the work so I'm only familiar with high level.
